README_B2ND_METALAYER.rst

```diff
@@ -52,12 +52,16 @@ Next, the `blockshape` section is meant to store the actual block shape info::
       |                +--[msgpack] int32
       +--[msgpack] int32
 
-Finally, the `dtype` section is meant to store the data type information.  Its representation follows the NumPy
-convention (as in `str(np.dtype)`; e.g. an `int32_t` dtype is represented as "int32")::
-
-    |---|--4 bytes---|--------------|
-    | db| dtype_len  | dtype_string |
-    |---|------------|--------------|
-      ^
-      |
-      +--[msgpack] str32
+Finally, the `dtype` section is meant to store the data type information::
+
+    |---|---|--4 bytes---|--------------|
+    | XX| db| dtype_len  | dtype_string |
+    |---|---|------------|--------------|
+      ^   ^
+      |   |
+      |   +--[msgpack] str32
+      +--[msgpack] positive fixint (7-bit integer). dtype_format; 0 means NumPy format.
+
+The 0 value for dtype_format means that that the dtype_string field follows the NumPy convention
```

Comment:
"that that" -> "that"

Reply:
Well spotted.  Fixed in ee660bcb (+ a problem with a malloc size)